yb_migrate/cmd/analyzeSchema.go

```diff
@@ -394,6 +396,10 @@ func checkDDL(sqlStmtArray [][]string, fpath string) {
 		} else if spc := alterTblSpcRegex.FindStringSubmatch(line[0]); spc != nil {
 			reportCase(fpath, "ALTER TABLESPACE not supported yet.",
 				"https://github.com/YugaByte/yugabyte-db/issues/1153", "", "TABLESPACE", spc[1], line[1])
+		} else if tbl := cLangRegex.FindStringSubmatch(line[0]); tbl != nil {
+			reportCase(fpath, "LANGUAGE C not supported yet.",
```

Comment:
can we imporve the message here? Instead of saying not supported, how about mentioning for non-superuser case.